dockers/llm.rag.service/common.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict, List
+
+
+def format_context(results: List[Dict[str, Any]]) -> str:
+    """Format search results into context for the LLM"""
+    context_parts = []
+
+    for result in results:
+        # TODO: make metadata keyes configurable
+        ticket_metadata = result.metadata
+        ticket_content = result.page_content
+
+        context_parts.append(
+            f"Key: {ticket_metadata['key']} | Status: {ticket_metadata['status']} - "
+            f"Type: {ticket_metadata['type']}\n"
+            f"Content: {ticket_content}...\n"
+        )
+
+    return "\n\n".join(context_parts)
+
+
+def get_answer_with_settings(question, retriever, client, model_id, max_tokens, model_temperature):
+    SYSTEM_PROMPT = """You are a specialized Jira ticket assistant. Format your responses following these rules:
```

Comment:
Given that we might start with tickets or issues from RedMine or Zendesk for our first pilot, could we replace "JIRA ticket" with something more generic?

Reply:
Shouldn't the context specifically say, "Answer the question only using the context provided"? Since we don't want the LLM to use any prior knowledge about any of the subject matter in its original training data?

Reply:
Now when this part is move out we can parametrize it. I would do it configurable.

> "Answer the question only using the context provided"
I don't know, but it sounds You may be right. Let's parametrize it in the follow up PR and verify. Now with the local setup possible it can be done easily.

Reply:
Let's do it in a separate PR.